spec/models/miq_queue_spec.rb

```diff
@@ -240,14 +240,6 @@
     after do
       Timecop.return
     end
```

Comment:
This context no longer contains any `it`s so the whole thing can be tossed... MORE ✂️